.github/workflows/appimage.yml

```diff
@@ -98,6 +100,65 @@ jobs:
         parent: false
         gzip: false
 
+  appimage-static-32bit:
+    name: static 32-bit for appimage
+    runs-on: buildjet-2vcpu-ubuntu-2204-arm
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event == 'pull_request_target' && github.event.pull_request.head.sha || github.ref }}
+    - uses: docker/setup-buildx-action@v2
+    - uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    # note: we run this way instead of using rdk-devenv as the job's container because of https://github.com/actions/runner/issues/2115 (js-based actions install incomaptible node on 32-bit)
+    - name: build server
+      run: docker run --rm -v $PWD:/rdk ghcr.io/viamrobotics/rdk-devenv:armhf-cache sh -c "cd /rdk && make server-static"
+    - name: build aix
+      run: docker run --rm -v $PWD:/rdk ghcr.io/viamrobotics/rdk-devenv:armhf-cache sh -c 'cd /rdk && GOBIN=`pwd`/bin/`uname -s`-`uname -m` go install -ldflags "-s -w" -tags osusergo,netgo github.com/Otterverse/aix@latest'
+    - name: permissions
+      run: sudo chown -R $USER bin
+    # note: we do this because uname sees the arm64 kernel underneath the armhf container
+    - name: fix platform detection
+      run: mv bin/`uname -s`-`uname -m` bin/`uname -s`-armv7l
+    - uses: actions/upload-artifact@v3
+      with:
+        name: appimage-static-32bit
+        path: |
+          bin/*/viam-server
+          bin/*/aix
+
+  # note: this is a separate job because 1) appimage-builder doesn't work on 32-bit, 2) setup-python doesn't work on arm64
```

Comment:
As with the bug-specific notes above, this one also feels like it _should_ be simpler, so I need to dig in slightly more and understand what these workarouns are fully doing. Will talk more tomorrow.